src/connectdlg.cpp

```diff
@@ -74,10 +74,12 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP,
     // central server address type combo box
     cbxCentServAddrType->clear();
     cbxCentServAddrType->addItem ( csCentServAddrTypeToString ( AT_DEFAULT ) );
```

Comment:
You could loop through an array containing the lists starting from here. 

Could this be done with a compiler macro or something like this: https://stackoverflow.com/questions/24392000/define-a-for-loop-macro-in-c#24392091